Server/service/stringService.js

```
@@ -0,0 +1,331 @@
+class StringService {
+  static SERVICE_NAME = "StringService";
+
+  constructor(translationService) {
+    if (StringService.instance) {
+      return StringService.instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)